test/chi.jl

```diff
@@ -1,2 +1,5 @@
 # see #1305
-@test isfinite(mean(Chi(1000)))
\ No newline at end of file
+@test isfinite(mean(Chi(1000)))
+
+# see #1497
+@test pdf(Chi(3), -.5) == 0.0
```

Comment:
It seems more natural to check the output of `logpdf` as this PR changes `logpdf`. Also good to check type stability for different types of `x` and `ν`:
```suggestion
for x in (-0.5, -1, -1.5f0), ν in (3, 3f0, 3.0)
    @test @inferred(logpdf(Chi(ν), x)) == -Inf
end 
```